contracts/MintShop1155.sol

```diff
@@ -42,13 +41,13 @@ contract MintShop1155 is Sweepable, ReentrancyGuard, IMintShop {
   bytes32 public constant LOCK_GLOBAL_LIMIT = keccak256("LOCK_GLOBAL_LIMIT");
 
   /// The public identifier for the right to manage whitelists.
-  bytes32 public constant WHITELIST = keccak256("WHITELIST");
+  bytes32 public constant override WHITELIST = keccak256("WHITELIST");
```

Comment:
Do we actually need this `override` modifier? What value are we actually overriding with this new constant?